services/integration/src/controllers/constitution/PreviousInformation.js

```diff
@@ -0,0 +1,30 @@
+const PreviousInformation = require('../../models/constitution/PreviousInformation.model');
```

Comment:
This directory should be named `previousInformation` or even shorter `previousInfo`

Reply:
i did changes